src/python/grpcio/grpc/_channel.py

```diff
@@ -194,25 +189,23 @@ def consume_request_iterator():
                   break
               else:
                 return
+        else:
+          return
     with state.condition:
       if state.code is None:
-        operations = (cygrpc.operation_send_close_from_client(),)
+        operations = (cygrpc.operation_send_close_from_client(_EMPTY_FLAGS),)
         call.start_batch(cygrpc.Operations(operations), event_handler)
         state.due.add(cygrpc.OperationType.send_close_from_client)
   thread = threading.Thread(target=consume_request_iterator)
   thread.start()
 
 
-class _Rendezvous(grpc.Future, grpc.Call, grpc.RpcError):
+class _Rendezvous(grpc.RpcError, grpc.Future, grpc.Call):
```

Comment:
`pylint`, actually - I'm pretty sure the interpreter was just fine with the order as it was.